keyboards/handwired/dactyl_manuform/5x7_2_6/keymaps/default/keymap.c

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com> Nikolai Grigoriev @ngrigoriev
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+enum layers {
+    _BASE,
+    _MAGIC,
+    _NUMPAD,
+    _FN
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_BASE] = LAYOUT_5x7_2_6(
+        // left hand
+        KC_GRV,         KC_1,       KC_2,    KC_3,   KC_4,   KC_5,   MO(_MAGIC),
+        KC_TAB,         KC_Q,       KC_W,    KC_E,   KC_R,   KC_T,   MO(_NUMPAD),
+        KC_LCTL,        KC_A,       KC_S,    KC_D,   KC_F,   KC_G,   _______,
+        OSM(MOD_LSFT),  KC_Z,       KC_X,    KC_C,   KC_V,   KC_B,
+                        KC_PGUP,  KC_PGDN,
+                                                KC_LCTL, LT(_FN, KC_ESC),
+                                                            KC_UP, KC_DOWN,
+                                                            KC_LCMD, KC_LALT,
+        // right hand
+                          KC_EQL,       KC_6,    KC_7,    KC_8,     KC_9,   KC_0,       KC_BSPC,
+                          KC_MINUS,     KC_Y,    KC_U,    KC_I,     KC_O,   KC_P,       KC_BSLS,
+                          KC_DEL,       KC_H,    KC_J,    KC_K,     KC_L,   KC_SCLN,    KC_QUOT,
+                                        KC_N,    KC_M,    KC_COMM,  KC_DOT, KC_SLSH,    OSM(MOD_RSFT),
+                                            KC_LBRC, KC_RBRC,
+                        KC_ENTER, KC_SPC,
+             KC_LEFT, KC_RIGHT,
+             KC_RCTL, KC_RALT
+    ),
```

Comment:
This should be side by side, not on top of each other.

Eg: 
```suggestion
    [_BASE] = LAYOUT_5x7_2_6(
        // left hand
        KC_GRV,         KC_1,       KC_2,    KC_3,   KC_4,   KC_5,   MO(_MAGIC),                  KC_EQL,       KC_6,    KC_7,    KC_8,     KC_9,   KC_0,       KC_BSPC,
        KC_TAB,         KC_Q,       KC_W,    KC_E,   KC_R,   KC_T,   MO(_NUMPAD),                  KC_MINUS,     KC_Y,    KC_U,    KC_I,     KC_O,   KC_P,       KC_BSLS,
        KC_LCTL,        KC_A,       KC_S,    KC_D,   KC_F,   KC_G,   _______,                  KC_DEL,       KC_H,    KC_J,    KC_K,     KC_L,   KC_SCLN,    KC_QUOT,
        OSM(MOD_LSFT),  KC_Z,       KC_X,    KC_C,   KC_V,   KC_B,                                KC_N,    KC_M,    KC_COMM,  KC_DOT, KC_SLSH,    OSM(MOD_RSFT),
                        KC_PGUP,  KC_PGDN,                                     KC_LBRC, KC_RBRC,
                                                KC_LCTL, LT(_FN, KC_ESC),            KC_ENTER, KC_SPC,
                                                            KC_UP, KC_DOWN,      KC_LEFT, KC_RIGHT,
                                                            KC_LCMD, KC_LALT,       KC_RCTL, KC_RALT
    ),
```

Reply:
First, this is how they are actually organized in QMK - https://docs.qmk.fm/features/split_keyboard#split-keyboard. The second half is actually a new set of rows below the first one. So, this layout matches the physical layout of the keyboard - how QMK sees it.

Of course, it does not match the way the humans see the keyboard, I agree. But I have two reasons why organizing it this way:

1. Avoiding long lines with many tabs in them. Long lines usually wrap in the editors, making them very difficult to manage. And once you start putting LALT_T(KC_XYZ) etc there, things get even worse.
2. It appears that there is no unified way - I see that some keymap.c files under handwired/ directory use "physical" layout while others use "human".

Do you think the point 1 at least is important? I can do it, but then, I think, I will need to change keyboard.json to reorder the layout items too.

Thanks!

Reply:
@drashna did you have a chance to review my arguments? Thanks.

Reply:
The handwired directory is a very bad example, as a lot of the boards there haven't been updated in ages, other than in mass updates. 

In general, yes the layout should be as the board appears to users.  In fact, in the split doc page, if you scroll down a bit more, the example explicitly is laid out this way: 
https://docs.qmk.fm/features/split_keyboard#layout-macro

Reply:
@drashna,

OK, for me it works either way, made the change to respect the convention. I have updated the JSON to reorder the layout entries accordingly. Tested with my layout, seems to be working correctly. Thanks for the review!